src/accumulator/proof.rs

```diff
@@ -1,6 +1,42 @@
+//! A proof is a data structure that proves that a given element is in the accumulator. It is used
```

Comment:
I'm going commit by commit. I think I addressed the comments here in a previous review comment